### PR TITLE
MODORDERS-937. Update RMB and vertx to the latest version

### PR DESCRIPTION
**Pull request by SerhiiNosko**
## Purpose
https://issues.folio.org/browse/MODORDERS-937

### CODE REVIEW

**Review by Dmitriy-Butramyou — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -51,15 +51,40 @@
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <ramlfiles_util_path>${ramlfiles_path}/raml-util</ramlfiles_util_path>
     <ramlfiles_acq_models_path>${ramlfiles_path}/acq-models</ramlfiles_acq_models_path>
-    <raml-module-builder.version>35.0.4</raml-module-builder.version>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <raml-module-builder.version>35.1.0</raml-module-builder.version>
+    <project.build.sourceEncoding>UTF-8 </project.build.sourceEncoding>
```
Comment (SerhiiNosko) [inline pom.xml:55]: space should be deleted

File: pom.xml [hunk 2]
```diff
@@ -51,15 +51,40 @@
...
+    <folio-di-support.version>2.0.1</folio-di-support.version>
+    <spring.version>6.0.2</spring.version>
+    <vertx.version>4.4.5</vertx.version>
+    <log4j.version>2.20.0</log4j.version>
+    <aspectj.version>1.9.20.1</aspectj.version>
+    <rest-assured.version>5.3.2</rest-assured.version>
+    <build-helper-maven-plugin.version>3.4.0</build-helper-maven-plugin.version>
+    <aspectj-maven-plugin.version>1.13.1</aspectj-maven-plugin.version>
+
+    <!--Maven dependencies-->
```
Comment (SerhiiNosko) [inline pom.xml:69]: propose to rename to maven plugin dependenices

File: pom.xml (hunk 1 above)
Comment (azizbekxm) [reply pom.xml]: Removed

File: pom.xml (hunk 2 above)
Comment (azizbekxm) [reply pom.xml]: Renamed

**Review by azizbekxm — APPROVED** [review]

**Pull request merged**